client/components/Pomodoro.tsx

```diff
@@ -37,7 +37,7 @@ export const Pomodoro: FC = () => {
             if (slackAuth) {
               window.bridge.slackSetProfile(slackAuth, {
                 text: mins === 1 ? `free in 1 min` : `free in ${mins} mins`,
-                emoji: ':tomato:',
+                emoji: ':eggplant:',
```

Comment:
Let the 24th Aug 2021 be known as the day the most important PR to github was ever made

Reply:
I didn't event know people could commit code to this 😂

Reply:
also the implications of this in your status have only just hit me 😂 😂